lib/zoom/constants.rb

```diff
@@ -92,5 +92,12 @@ class Constants
                    'pro' => 2,
                    'corp' => 3
                   }.freeze # no default
+
+    USER_CREATE_TYPES = {
+                          'User will get an email sent from Zoom. There is a confirmation link in this email. User will then need to click this link to activate their account to the Zoom service. The user can set or change their password in Zoom.' => 'create',
```

Comment:
Just a minor style thing, I would swap the keys & values here so that the shorter argument is the key. 

```suggestion
  'create' =>  'User will get an email sent from Zoom. There is a confirmation link in this email. User will then need to click this link to activate their account to the Zoom service. The user can set or change their password in Zoom.',
```

Reply:
I fixed. Thanks for your review!